### PR TITLE
Minor simplifications and code cleanup.

### PR DESCRIPTION
**Pull request by nickmshelley**

### CODE REVIEW

**Comment by nickmshelley** [comment]
@hiltonc I don't have any experience with Travis, but it looks like it's not using the latest Xcode. How do we change that?

**Comment by nickmshelley** [comment]
Never mind, found travis.yml.

File: LDSContent/Catalog.swift [hunk 1]
```diff
@@ -38,17 +38,15 @@ public class Catalog {
         return schemaVersion
     }()
     
-    let db: Connection!
-    let noDiacritic: ((Expression<String>) -> Expression<String>)!
+    let db: Connection
```
Comment (hiltonc) [inline LDSContent/Catalog.swift]: yes!

File: LDSContent/Session.swift [hunk 2]
```diff
@@ -50,8 +50,8 @@ public class Session: NSObject {
 
 extension Session: NSURLSessionDelegate {
     
-    public func URLSession(session: NSURLSession, didReceiveChallenge challenge: NSURLAuthenticationChallenge, completionHandler: (NSURLSessionAuthChallengeDisposition, NSURLCredential?) -> Void) {
-        completionHandler(.UseCredential, NSURLCredential(forTrust: challenge.protectionSpace.serverTrust!))
+    func URLSession(session: NSURLSession, didReceiveChallenge challenge: NSURLAuthenticationChallenge, completionHandler: (NSURLSessionAuthChallengeDisposition, NSURLCredential?) -> Void) {
+        completionHandler(.UseCredential, challenge.protectionSpace.serverTrust.flatMap { NSURLCredential(forTrust: $0) })
```
Comment (hiltonc) [inline LDSContent/Session.swift]: The ! had a certain elegance.

**Comment by hiltonc** [comment]
Yeah, just push a change to travis.yml and it will rebuild.

**Comment by hiltonc** [comment]
👍

File: LDSContent/Session.swift (hunk 2 above)
Comment (nickmshelley) [inline LDSContent/Session.swift]: So does `flatMap` :)

Comment (hiltonc) [inline LDSContent/Session.swift]: I was kidding!

**Pull request merged**